src/components/CompareResults/RevisionRowExpandable.tsx

```diff
@@ -79,14 +83,18 @@ function RevisionRowExpandable(props: RevisionRowExpandableProps) {
             </div>
           )}
         </div>
-        <div className={`${styles.bottomSpace}`}>
-          <b>Mean Difference</b>: {deltaPercent}%{' '}
+        <div>
+          <b>Difference of means</b>: {deltaPercent}%{' '}
           {newIsBetter ? 'better' : 'worse'} ({delta} {deltaUnit})
         </div>
+        <div>
+          <b>Difference of medians</b>: {(newMedian - baseMedian).toFixed(2)} (
+          {(((newMedian - baseMedian) / baseMedian) * 100).toFixed(2)}%)
+        </div>
```

Comment:
It's a bit weird when one of the values is zero, so I'd do something like this:
```js
{newMedian && baseMedian ? (
  <div>
     ...
  </div>
) : null}
```

Reply:
It would be good to have the percentage value first and the absolute value next, like for the means, otherwise it's difficult to read. And also to have the unit for the absolute value like for the means.

It would be easier to do the computation before the JSX part so that you just have to use the values here.

It would also be good to compute if it's worse/better using the `lower_is_better` property, again to do exactly like the difference of the means, for an easier read.